src/main/java/com/artipie/asto/Remaining.java

```diff
@@ -29,19 +29,34 @@
  * Remaining bytes in a byte buffer.
  * @since 0.13
  */
-public class Remaining {
+public final class Remaining {
```

Comment:
@g4s8 Shouldn't this class represent a collection of bytes? Why not making it implement `Iterable` or something like that?

Reply:
@paulodamaso it would be not good for performance, we're going to handle few thousands of requests per second, so casting `byte` to `Byte` will be inefficient operation for CPU and memory (`Byte` takes about 4x more memory than `byte`)